src/Tools/Microsoft.VisualStudio.Templates.Editorconfig.Wizard/Generator/RoslynEditorConfigFileGenerator.cs

```diff
@@ -1,56 +1,22 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
 
 using EnvDTE80;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Options;
-using Microsoft.VisualStudio.ComponentModelHost;
-using Microsoft.VisualStudio.LanguageServices;
-using Microsoft.VisualStudio.RpcContracts.FileSystem;
 using Microsoft.VisualStudio.Shell;
-using Microsoft.VisualStudio.Templates.Editorconfig.Wizard.Logging.Kinds;
-using Microsoft.VisualStudio.Templates.Editorconfig.Wizard.Logging.Messages;
 using System;
-using System.Collections.Immutable;
-using System.IO;
 using System.Reflection;
 using static Microsoft.VisualStudio.Templates.Editorconfig.Wizard.Logging.Logger;
+using Microsoft.CodeAnalysis.ExternalAccess.EditorConfigGenerator.Api;
 
 namespace Microsoft.VisualStudio.Templates.Editorconfig.Wizard.Generator;
 
 public class RoslynEditorConfigFileGenerator
 {
-    private const BindingFlags PublicBindingFlags = BindingFlags.Instance | BindingFlags.Static | BindingFlags.Public;
-    private const BindingFlags NonPublicBindingFlags = BindingFlags.Instance | BindingFlags.Static | BindingFlags.NonPublic;
-
     public RoslynEditorConfigFileGenerator(DTE2 dte)
     {
         try
         {
-            string? basePath = Path.GetDirectoryName(dte.FullName);
```

Comment:
Great to see so much being deleted here. And wow this code was scary and broken...